java/src/main/java/com/saucelabs/saucebindings/SaucePlatform.java

```diff
@@ -7,6 +7,7 @@
 import java.util.Set;
 
 public enum SaucePlatform {
+    LINUX("Linux"),
```

Comment:
We have to do this, don't we @titusfortner

Reply:
Yeah, it's required for Headless, unless we want to handle headless differently via Session (which I'm also ok with)

Reply:
I feel like we either have to add this now or remove US_EAST support from Session.

Reply:
Nah let's keep it like this for now.